pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -57,6 +58,14 @@ var updateBackoff = wait.Backoff{
 	Jitter:   1.0,
 }
 
+var blacklistKubeletConfigurationFields = []string{
```

Comment:
nit: some documentation around why might come in handy here or in docs.

Reply:
great idea

Reply:
Added